File: boilerplate/backend/node-express/config/emailpassword.ts
import EmailPassword from "supertokens-node/recipe/emailpassword";
import Session from "supertokens-node/recipe/session";
import { TypeInput } from "supertokens-node/types";
import Dashboard from "supertokens-node/recipe/dashboard";

export const SuperTokensConfig: TypeInput = {
    supertokens: {
        // this is the location of the SuperTokens core.
        connectionURI: "https://try.supertokens.com",
    },
    appInfo: {
        appName: "SuperTokens Demo App",
        apiDomain: "http://localhost:3001",
        websiteDomain: "http://localhost:3000",
    },
    // recipeList contains all the modules that you want to
    // use from SuperTokens. See the full list here: https://supertokens.com/docs/guides
    recipeList: [
        EmailPassword.init(),
        Session.init(),
        Dashboard.init({
            apiKey: "supertokens_is_awesome",
        }),
    ],
};

File: boilerplate/backend/python-flask/config/emailpassword.py
from supertokens_python.recipe import emailpassword, session
from supertokens_python import (
    InputAppInfo,
    SupertokensConfig,
)

supertokens_config=SupertokensConfig(connection_uri="https://try.supertokens.io")

app_info=InputAppInfo(
    app_name="Supertokens",
    api_domain="http://localhost:3001",
    website_domain="http://localhost:3000",
)

framework="flask"

recipe_list=[
    session.init(),
    emailpassword.init()
]